src/main/java/io/debezium/connector/jdbc/dialect/postgres/ConnectBytesTypePostgres.java

```diff
@@ -0,0 +1,22 @@
+package io.debezium.connector.jdbc.dialect.postgres;
+
+import io.debezium.connector.jdbc.type.connect.ConnectBytesType;
+import org.apache.kafka.connect.data.Schema;
+import org.hibernate.query.Query;
+
+import java.nio.ByteBuffer;
+
+public class ConnectBytesTypePostgres extends ConnectBytesType {
```

Comment:
I'll suggest to create the `BytesType` and extend `AbstractType` to be uniform of what already did for others database (i.e MySQL).